message_sender/tests.py

```diff
@@ -420,6 +473,65 @@ def test_create_outbound_data_with_channel_unknown(self):
                                     content_type='application/json')
         self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
 
+    def test_create_outbound_without_recipient(self):
+
+        post_outbound = {
+            "delivered": "false",
+            "metadata": {}
+        }
+        response = self.client.post('/api/v1/outbound/',
+                                    json.dumps(post_outbound),
+                                    content_type='application/json')
+        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
+
+    @responses.activate
+    def test_create_outbound_identity_only(self):
```

Comment:
The `test_create_outbound_data` and `test_create_outbound_data_simple` both creates outbounds with only the `to_addr` populated.